modules/setup/application/clicommands/ConfigCommand.php

```diff
@@ -119,6 +121,7 @@ public function directoryAction()
      *
      *  icingacli setup config webserver nginx \
      *    --root=/usr/share/icingaweb2/public \
+     *    --enable-fpm
```

Comment:
Above the help for this switch says it's always enabled for nginx and now it's added here? :eyes:

Reply:
Thanks, oversight. Fixed.